src/odemis/acq/test/millmng_test.py

```diff
@@ -407,7 +410,6 @@ def test_milling_settings_from_yaml(self):
         f = mill_features(millings, self.sites, self.feature_post_status, self.acq_streams, self.ion_beam,
                           self.sed, self.stage, self.aligner)
 
-        time.sleep(13)  # Long enough to make sure it's started
```

Comment:
why the sleeps are removed here?

Reply:
because the mill_features already starts the future, after 13 seconds the future had already stopped so that's why the test case failed

Reply:
13 s was probably really long, but now you risk having test case failures because the future might not have started yet... I think a `sleep(0.1)` would be useful here.

Reply:
We tested it and it was failing with a sleep of 0.1s, it would pass with a sleep of 0.00001s but I don't know if it makes sense to add that.

Reply:
Well, then it sounds to me like something is really broken. It should be milling. Milling should take several seconds.